src/tree/assess/TreeAssessor.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Analyzes a paper by doing a list of assessments on a tree representation of a text and its metadata.
+ * Aggregates the scores on each individual assessment into an overall score.
+ *
+ * This score can represent anything from the readability to the SEO of the given text and metadata.
+ *
+ * @memberOf module:tree/assess
+ */
+class TreeAssessor {
+	/**
+	 * Creates a new assessor.
+	 *
+	 * @param {Object} options                                             Assessor options.
+	 * @param {Jed} options.i18n                                           A Jed object to use for translations.
+	 * @param {module:tree/research.TreeResearcher} options.researcher     Supplies the assessments with researches and their (cached) results.
+	 * @param {module:tree/assess.ScoreAggregator} options.scoreAggregator Aggregates the scores on the individual assessments into one overall score.
+	 * @param {module:tree/assess.Assessment[]} [options.assessments]      The list of assessments to apply.
+	 */
+	constructor( options ) {
+		/**
+		 * A Jed object to use for translations.
+		 * @type {Jed}
+		 */
+		this.i18n = options.i18n;
+		/**
+		 * Supplies the assessments with researches and their (cached) results.
+		 * @type {module:tree/research.TreeResearcher}
+		 */
+		this.researcher = options.researcher;
```

Comment:
Since the researcher is (for now) only used inside of the `assess` function, I would vote for giving it as an extra parameter in the `assess` function. Rather than giving it as a constructor argument.

Reply:
We (@igorschoester and I) discussed this with @atimmer and we decided to give it as a constructor argument in the `TreeAssessor` as well as the `Assessment`.

This is conceptually the right thing to do, since we only have one researcher available at any given time.